tests/go/kgo-verifier/go.mod

```diff
@@ -17,6 +18,9 @@ require (
 	golang.org/x/time v0.9.0
 )
 
+// temporary workaround for CORE-14849/CORE-13402
+replace github.com/twmb/franz-go v1.20.2 => github.com/travisdowns/franz-go v1.20.2-CORE-14849-fix
```

Comment:
Can we fork this into redpanda? Having a private one feels odd? Not sure what we did in the past.

Reply:
After talking with @twmb he has pushed a PR upstream which makes this configurable and for now we can use that, so I propose this new PR which will supersede this one:

https://github.com/redpanda-data/redpanda/pull/28854